hoodie-client/src/main/java/com/uber/hoodie/config/HoodieIndexConfig.java

```diff
@@ -54,8 +54,35 @@
   public static final String HBASE_ZKPORT_PROP = "hoodie.index.hbase.zkport";
```

Comment:
General comment : It's good to have configs exposed for tuning parameters but there are a lot of configs here. How are users even supposed to use these configs ? For eg., how does one determine what value to use for a parameters such as : HBASE_SLEEP_MS_PUT_BATCH_PROP / HBASE_SLEEP_MS_GET_BATCH_PROP ?

Reply:
They are already added, checked it again, do you see any specific params missed?

Reply:
As we talked offline, removing the SLEEP configs would make sense here. Also, please check the possibility of removing any other configs which may not be necessary.

Reply:
Done

Reply:
should we be thinking about breaking up this file into HoodieHBaseIndexConfig?

Reply:
I'm in favor of this, didn't want to introduce new config classes unnecessarily but with so many configs for hbase, it makes sense.

Reply:
Moved it to HoodieHBaseIndexConfig